ansible/roles/routemgmt/files/gwapis_views.json

```diff
@@ -0,0 +1,16 @@
+{
+  "_id": "_design/gwapis",
+  "views": {
+    "routes-by-path": {
+      "map": "function (doc) {\n  var PATHSEP = \"/\";\n\n  if (doc.namespace && doc.gatewayPath) {\n    emit([doc.namespace, doc.gatewayPath], {\"gatewayMethod\":doc.gatewayMethod, \"gatewayPath\":doc.gatewayPath, \"action\":doc.action, \"route-collection\":doc[\"route-collection\"], \"apidoc\":doc.apidoc});\n  }\n}"
```

Comment:
@markusthoemmes - at the moment, no.  But for performance, I can imagine views that return a subset of data from the docs